src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -392,6 +402,43 @@ class HaskellCompletionContributor extends CompletionContributor {
     waitForResult(lookupElements)
   }
 
+  private def getIdsFromAllExportedModules(project: Project) = {
```

Comment:
It only takes them for project files and we have ghci/intero command for it :smile: